resources/js/Shared/Layout/SearchPanelSkeleton.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="mx-auto mt-12 w-3/4 animate-pulse p-4">
+    <p class="mb-3 mr-2 h-2 w-24 rounded bg-gray-400 font-bold" />
+    <input class="mb-2 w-full rounded-md border-0 bg-gray-200 py-2 pl-3 pr-10 text-blumilk-700 shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-blumilk-500 sm:leading-6">
+    <p class="mb-3 mr-2 mt-5 h-2 w-16 rounded bg-gray-400 font-bold" />
+    <input class="mb-8 w-full rounded-md border-0 bg-gray-200 py-2 pl-3 pr-10 text-blumilk-700 shadow-sm ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-inset focus:ring-blumilk-500 sm:leading-6">
+
+
+    <ul class="divide-y divide-gray-300" role="list">
+      <li class="flex items-start justify-between gap-x-6 py-5">
+        <div class="flex flex-auto items-center">
+          <i class="hr flat flag huge shrink-0" />
+          <div class="ml-4 flex flex-col justify-start">
+            <p class="mb-3 mr-2 h-2 w-24 rounded bg-gray-400 font-bold" />
+            <p class="h-2 w-12 rounded bg-blumilk-400 text-xs font-semibold" />
+          </div>
+        </div>
+        <div class="flex w-full flex-row-reverse flex-wrap items-center">
+          <div class="m-1 flex h-7 w-10 shrink-0 items-center justify-center rounded-md border border-zinc-300 bg-zinc-300 p-1"
+               style="background-color: #80ff80"
```

Comment:
Do we really need `style= ...` attribute? We can write in `class` something like `bg-green-300`. Albo we should avoid using color in hex - default palette for this component is enough, but if your really, really want, you can make custom names for this color in `tailwind.config.js`

Reply:
These color in hex are used just to imitate the colors of the providers logos. It is used only in that skeleton and doesn't need global name in config.

![image](https://github.com/blumilksoftware/escooters/assets/92044526/d62416fb-c23e-48cc-9dbb-1eef3c0f0450)